WEBKIT-MEDIA-STUBS.md

```diff
@@ -0,0 +1,51 @@
+# Web Testing
+
+The web project needs WebKit and Media shared libraries to run tests.
+
+These can be supplied in a number of ways. See sections below.
+
+## Compiled from source
+
+Specify these Gradle properties to enable building of WebKit and Media libraries from source:
+
+    -PCOMPILE_WEBKIT=true -PCOMPILE_MEDIA=true
+
+Note that these require additional build tooling and take some time to build.
+
+If you are not actively working on these sources, you may want to cache the output by copying it to one of the folders mentioned below.
+
+
+## Cached libraries
+
+You can manually place WebKit and Media shared libraries in these folders:
+
+* Unix libraries (*.so or *.dylib files)
+````
+    $projectDir/../caches/sdk/lib
```

Comment:
I know the caches have been at this location in the past, but I find it intrusive if `$projectDir/..` is used, so my preference would be to have it at `$projectDir/caches` . But maybe there is a good reason for using .. ?

Reply:
This is just documenting current behavior. We have many build scripts rely on this.